src/FSLibrary/MissionSorobanClassicMixed.fs

```diff
@@ -0,0 +1,84 @@
+// Copyright 2024 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+module MissionSorobanClassicMixed
+
+open StellarCoreSet
+open StellarCorePeer
+open StellarMissionContext
+open StellarFormation
+open StellarStatefulSets
+open StellarSupercluster
+open StellarNetworkData
+open StellarCoreHTTP
+
+let sorobanClassicMixed (context: MissionContext) =
+
+    let context =
+        { context with
+              numAccounts = 2000
+              numTxs = 2000
+              txRate = 20
+              skipLowFeeTxs = true }
+
+    let sorobanCoreSet =
+        MakeLiveCoreSet
+            "soroban"
+            { CoreSetOptions.GetDefault context.image with
+                  quorumSet = CoreSetQuorum(CoreSetName "core")
```

Comment:
I think you want `CoreSetQuorumList([ CoreSetName "soroban"; CoreSetName "classic" ])` here ("core" core set doesn't exist in this mission)